### PR TITLE
Add toast notifications

### PR DESCRIPTION
**Pull request by Pawci0**
**NOTICE:** to be merged only after #27

### CODE REVIEW

File: pandemania/src/App.jsx [hunk 1]
```diff
@@ -33,6 +35,7 @@ function App() {
 							</Switch>
 						</div>
 					</Router>
+					<ToastContainer position="bottom-center" />
```
Comment (jurczewski) [inline pandemania/src/App.jsx:39]: Should it be lazy loaded?

Comment (Pawci0) [reply pandemania/src/App.jsx:39]: React.lazy doesn't support importing non-default components

File: pandemania/src/api/Toasts.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { toast } from 'react-toastify';
+
+export const apiErrorToast = () => {
+	toast.error('Covid API id down, please try again later', { toastId: 'api-error' });
```
Comment (jurczewski) [inline pandemania/src/api/Toasts.js]: "Is" instead "id"
And maybe covid 19 instead of covid.

File: pandemania/src/api/Toasts.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+};
+
+export const loginSuccessToast = () => {
+	toast.success('Successfully logged in!', { toastId: 'login' });
```
Comment (jurczewski) [inline pandemania/src/api/Toasts.js:8]: Consider ending every toast with punctuation mark.

Comment (Pawci0) [reply pandemania/src/api/Toasts.js:8]: added "!" to every toast

File: pandemania/src/api/Toasts.js (hunk 2 above)
Comment (Pawci0) [reply pandemania/src/api/Toasts.js]: fixed

**Pull request merged**